### PR TITLE
Update fr.json

### PR DESCRIPTION
**Pull request by Zazou89**

### CODE REVIEW

**Comment by jliddev** [comment]
Something went wrong with your commit, the json is all jacked up.

**Review by SteadEXE — COMMENTED** [review]
There is a typo line 111 "ID de l'addon copé" -> "ID de l'addon copié"

File: wowup-electron/src/assets/i18n/fr.json [hunk 1]
```diff
@@ -107,13 +107,13 @@
     "ADDON_DETAILS": {
       "ADDON_ID_PREFIX": "Addon ID:",
       "BY_AUTHOR": "Par {authorName}",
-      "CHANGELOG_TAB": "Changelog",
-      "COPY_ADDON_ID_SNACKBAR": "Addon ID copied to clipboard",
-      "COPY_ADDON_ID_TOOLTIP": "Copy addon ID to clipboard",
+      "CHANGELOG_TAB": "Notes de mise à jour ",
+      "COPY_ADDON_ID_SNACKBAR": "ID de l'addon copé dans le presse-papier",
```
Comment (SteadEXE) [inline wowup-electron/src/assets/i18n/fr.json:111]: Typo here: "copé" should be "copié"

**Comment by Zazou89** [comment]
Thanks @SteadEXE, I'll try to make a new pull request. You can close this one.

**Pull request closed without merging**